src/neo/Wallets/Wallet.cs

```diff
@@ -391,14 +391,18 @@ public long CalculateNetworkFee(StoreView snapshot, Transaction tx)
                 {
                     var contract = NativeContract.ContractManagement.GetContract(snapshot, hash);
                     if (contract is null) continue;
+                    var md = contract.Manifest.Abi.GetMethod("verify", 0);
```

Comment:
```suggestion
                    var md = contract.Manifest.Abi.GetMethod("verify", -1);
```

Reply:
I think the wallet support `0` parameter only. Because no `InvocationScript` is provided.

Reply:
Just a side note, but we use `verify` with parameter (a signature) in the Notary contract (#1573). But this can be accounted for when/if implementing support for it.